docs/how-to-setup-wsl.md

```diff
@@ -126,6 +128,65 @@ Now that you have installed the pre-requisites, follow [our local setup guide](h
 >
 > Please note, at this time the setup for Cypress tests (and related GUI needs) is a work in progress. You should still be able to work on most of the codebase.
 
+## Optimize Windows and WSL
+
+   > [!NOTE]
+   >
+   > The following tips were collected from across the web and have not gone through vigorous testing. Your mileage may vary.
+
+### Adjust processer scheduling for background services
+
+This may reduce incidents of Docker containers crashing due to lack of resources.
+
+Open the System Properties control panel by pressing <kbd>Win + R</kbd> and entering `sysdm.cpl`
+
+![Run dialog](images/wsl/run-sysdm.png)
```

Comment:
Wrap images in a collapsed details view. 

Example:
https://github.com/freeCodeCamp/freeCodeCamp/blob/5c5889dd3a3cd6ad2eb8cc1fef037e64e5c15abb/docs/how-to-setup-freecodecamp-locally.md?plain=1#L127-L133

Note that you must do this in a separate PR to use the `<img>` tag and upload the images first.

Reply:
You mean like this? https://github.com/freeCodeCamp/freeCodeCamp/pull/52239

Reply:
Yes, that is fine.